packages/storybook-builder-parcel/gen-preview-modern.js

```diff
@@ -7,9 +7,16 @@ const {
 } = require("@storybook/core-common");
 const { logger } = require("@storybook/node-logger");
 const { promise: glob } = require("glob-promise");
+const isMingw = require('is-mingw');
```

Comment:
changes in this file were to fix some problem with Windows development

Reply:
can we not just do `process.platform === 'win32'`?

Reply:
heh yeah, looks like we did that dependency locally, probably easier to check the process